resqpy/olio/vector_utilities.py

```diff
@@ -862,6 +862,62 @@ def points_in_triangles_aligned(nx: int, ny: int, dx: float, dy: float, triangle
     return triangles_points
 
 
+@njit
+def triangle_box(tri):
+    x_values = tri[:, 0]
+    y_values = tri[:, 1]
+
+    def min_max(x):
+        maximum = x[0]
+        minimum = x[0]
+        for i in x[1:]:
```

Comment:
We know there are exactly three rows in the tri array, so better to have two if blocks and avoid the overhead of the for loop I think.